### DISCUSSION TITLE
Serve “Tonight’s Dinner” from the meal plan to a webpage

### DISCUSSION THREAD

**Discussion by hodaddy4** [TandoorRecipes/recipes#1704] [General]
Hello, I am really enjoying Tandoor Recipes… I have it installed in a docker on my Unraid Server. What I would love to do is be able to have my kitchen family DakBoard pull “Tonight’s Dinner” and display it on our kitchen display. One limiting factor is that DakBoard requires a HTTPS imbed link… for security reasons. It would be great if I could link DakBoard with TandoorRecipes. One other option is doing some sort of JSON thing, but I have no idea how to do that lol. Even if maybe Tandoor could automatically export todays meal plan into a pdf or an image, maybe I could simply display that inside DakBoard as an alternative option. Any help is greatly appreciated

**Comment by vabene1111** [comment]
this is certainly a good idea and something that will likely be implemented in the future. For now i think the best solution would be some kind of redirect system that pulls "todays" dinner from the api.

**Comment by jaud7of8** [comment]
Along this line, is it possible to programmaticly export the calendar of food, to be imported to another calendar, such as nextcloud and be used to display on a calendar in magicmirror?  
Thanks

**Reply by vabene1111** [reply]
Thanks for the idea, features like this are already planned for future releases

**Comment by wendysmenu6-a11y** [comment]
You could integrate Tandoor Recipes with DakBoard by creating a secure HTTPS link or exporting the “Tonight’s Dinner” meal plan as a PDF/image for display, which avoids JSON complexity. According to Wikipedia, meal planning software often supports multiple export formats, making it easier to adapt to various display tools. For those exploring digital menu inspirations, the [wendys.com menu](https://wendys-menu-prices.com/) is a great example of how food options can be presented in a clear, accessible way.

**Comment by proseo2050** [comment]
You could achieve this by using Tandoor Recipes’ API to output “Tonight’s Dinner” in JSON format, then host that data via a small HTTPS-enabled service or reverse proxy so DakBoard can embed it securely per HTTPS standards. Alternatively, schedule a script to auto-generate a daily PDF/image from your meal plan and place it in a web-accessible HTTPS directory for DakBoard to display. For those managing home displays alongside menu planning, checking arbys-menu-prices can be a fun way to[ mix home-cooked ](https://arbys-menu-prices.com/)and takeout options.
